### PR TITLE
Attempt to clarify slots discussion

### LINKED ISSUE #301: Some confusion about the 2nd paragraph in the Composite Device (3.5) section
> But other slots can communicate with the main slot by the links between them inside the router.

I am not certain what this sentence is trying to tell me in this context?

### PR DESCRIPTION
**Pull request by dthaler**
Fixes #301

Signed-off-by: Dave Thaler <dthaler@ntdev.microsoft.com>

### CODE REVIEW

**Review by thomas-fossati — APPROVED** [review]
lgtm

File: draft-ietf-rats-architecture.md [hunk 1]
```diff
@@ -498,10 +498,10 @@ For example, a carrier-grade router consists of a chassis and multiple slots.
 The trustworthiness of the router depends on all its slots' trustworthiness.
 Each slot has an Attesting Environment such as a TEE collecting the
 Claims of its boot process, after which it generates Evidence from the Claims.
-Among these slots, only a main slot can communicate with the Verifier
-while other slots cannot. But other slots can communicate with the main
-slot by the links between them inside the router. So the main slot collects
-the Evidence of other slots, produces the final Evidence of the whole router and
+Among these slots, only a "main" slot can communicate with the Verifier
+while other slots cannot, but other slots can communicate with the main
+slot across the links between slots inside the router. The main slot thus collects
```
Comment (mcr) [inline draft-ietf-rats-architecture.md:503]:
```suggestion
while other slots cannot, but other slots can communicate with the "main"
slot across the links between slots inside the router. The "main" slot thus collects
```

Comment (mcr) [reply draft-ietf-rats-architecture.md:503]: I'm not sure why "main" is quoted, but should it be consistent?

Comment (dthaler) [reply draft-ietf-rats-architecture.md:503]: It was quoted the first time as being defined in context, and then used unquoted thereafter.  See for example https://blog.apastyle.org/apastyle/2013/02/quotation-mark-uses-other-than-quotes.html which quotes the guidance:
```
To introduce a word or phrase used as an ironic comment, as slang, or as an invented or coined expression. Use quotation marks the first time the word or phrase is used; thereafter, do not use quotation marks. (p. 91)
```
(Here "main" router is an invented or coined expression)

**Comment by dthaler** [comment]
I see PR #296 contains a different rewrite of this text.  We should compare them and decide which one to take, or whether to take some of each.

**Pull request closed without merging**